src/view/card-film.js

```diff
@@ -1,15 +1,17 @@
-export const createCardFilm = () => {
+export const createCardFilm = (filmCard) => {
+  const {filmName, poster, description, raiting, genre, comments, year, duration} = filmCard;
+  const yearCard = year.slice(-4);
   return `<article class="film-card">
-  <h3 class="film-card__title">Title</h3>
-  <p class="film-card__rating">0</p>
+  <h3 class="film-card__title">${filmName}</h3>
+  <p class="film-card__rating">${raiting}</p>
   <p class="film-card__info">
-    <span class="film-card__year">0000</span>
-    <span class="film-card__duration">0m</span>
-    <span class="film-card__genre">genre</span>
+    <span class="film-card__year">${yearCard}</span>
+    <span class="film-card__duration">${duration}</span>
+    <span class="film-card__genre">${genre[0]}</span>
   </p>
-  <img src="./images/posters/santa-claus-conquers-the-martians.jpg" alt="" class="film-card__poster">
-  <p class="film-card__description">description</p>
-  <a class="film-card__comments">0 comments</a>
+  <img src="./images/posters/${poster}" alt="" class="film-card__poster">
+  <p class="film-card__description">${description}</p>
+  <a class="film-card__comments">${comments.text.length}</a>
```

Comment:
Слово comments не удаляй
```suggestion
  <a class="film-card__comments">${comments.text.length} comments</a>
```